lib/ansible/modules/cloud/amazon/ec2_elb_lb.py

```diff
@@ -477,7 +477,12 @@ def __init__(self, module, name, listeners=None, purge_listeners=None,
         self.changed = False
         self.status = 'gone'
         self.elb_conn = self._get_elb_connection()
-        self.elb = self._get_elb()
+
+        try:
+            self.elb = self._get_elb()
+        except boto.exception.BotoServerError as e:
+            module.fail_json(msg=str(e))
```

Comment:
I've added messages and exception details. Since `BotoServerError` doesn't provide a dictionary with error details, I wasn't able to use `camel_dict_to_snake_dict`.